sword-server/src/main/java/edu/unc/lib/dl/cdr/sword/server/managers/ContainerManagerImpl.java

```diff
@@ -48,58 +36,10 @@
  */
 public class ContainerManagerImpl extends AbstractFedoraManager implements ContainerManager {
 
-    private static Logger log = LoggerFactory.getLogger(ContainerManagerImpl.class);
-
-    //    private DigitalObjectManager digitalObjectManager;
-    private UIPProcessor uipProcessor;
-    //    private ManagementClient managementClient;
-    private DepositReportingUtil depositReportingUtil;
-
     private DepositReceipt updateMetadata(String editIRI, Deposit deposit, AuthCredentials auth,
             SwordConfiguration config, UpdateOperation operation) throws SwordError, SwordServerException,
     SwordAuthException {
-        PID targetPID = extractPID(editIRI, SwordConfigurationImpl.EDIT_PATH + "/");
-
-        SwordConfigurationImpl configImpl = (SwordConfigurationImpl) config;
-
-        if (!hasAccess(auth, targetPID, Permission.editDescription, configImpl)) {
-            throw new SwordError(ErrorURIRegistry.INSUFFICIENT_PRIVILEGES, 403,
-                    "Insufficient privileges to update metadata for " + targetPID.getPid());
-        }
-
-        AtomPubMetadataUIP uip;
-        try {
-            uip = new AtomPubMetadataUIP(targetPID, auth.getUsername(), operation, deposit.getSwordEntry().getEntry());
-        } catch (UIPException e) {
-            log.warn("An exception occurred while attempting to create metadata UIP for " + targetPID.getPid(), e);
-            throw new SwordError(ErrorURIRegistry.INGEST_EXCEPTION, 500,
-                    "An exception occurred while attempting to create metadata UIP for " + editIRI, e);
-        }
-
-        try {
-            uipProcessor.process(uip);
-        } catch (UpdateException e) {
-            if (e.getCause() instanceof AuthorizationException) {
-                throw new SwordError(ErrorURIRegistry.INSUFFICIENT_PRIVILEGES, 403,
-                        "Failed to authorize update metadata operation", e);
-            }
-            throw new SwordError(ErrorURIRegistry.UPDATE_EXCEPTION, 500,
-                    "An exception occurred while attempting to update object " + targetPID.getPid(), e);
-        } catch (UIPException e) {
-            log.warn("Failed to process UIP for " + targetPID.getPid(), e);
-            throw new SwordError(ErrorURIRegistry.UPDATE_EXCEPTION, 500,
-                    "A problem occurred while attempting to perform the requested update operation on "
-                            + editIRI, e);
-        }
-
-        DepositReceipt receipt = new DepositReceipt();
-        receipt.setLocation(new IRI(editIRI));
-        receipt.setEmpty(true);
-
-        // Update the objects in progress status
-        this.setInProgress(targetPID, deposit, receipt);
-
-        return receipt;
+        throw new NotImplementedException("Operation not supported");
```

Comment:
this invites the Liar's Paradox. "I am not implemented," says the implementation ;-)

Reply:
Are you calling Boxy a liar?

There's actually a OperationNotSupportedException in the java core libraries, but its got some weird connotations going on with it https://docs.oracle.com/javase/8/docs/api/index.html?javax/naming/OperationNotSupportedException.html